docs/configure-artifact-repository.md

```diff
@@ -410,30 +416,31 @@ In addition to managed identities, Argo workflows also support authentication us
 
 ### Using Azure Shared Access Signatures (SAS)
 
-If you do not wish to use an access key, you may also use a [shared access signature (SAS)](https://learn.microsoft.com/en-us/azure/storage/common/storage-sas-overview?toc=%2Fazure%2Fstorage%2Fblobs%2Ftoc.json&bc=%2Fazure%2Fstorage%2Fblobs%2Fbreadcrumb%2Ftoc.json).
-Create an Azure Storage account and a container within that account.
-There are several ways to accomplish this, including the [Azure Portal](https://portal.azure.com) or the [CLI](https://docs.microsoft.com/en-us/cli/azure/).
+> v3.6 and after
+
+You can also use a [Shared Access Signature (SAS)](https://learn.microsoft.com/en-us/azure/storage/common/storage-sas-overview?toc=%2Fazure%2Fstorage%2Fblobs%2Ftoc.json&bc=%2Fazure%2Fstorage%2Fblobs%2Fbreadcrumb%2Ftoc.json).
 
-1. Retrieve the blob service endpoint for the storage account. For example:
+1. Retrieve the blob service endpoint for the storage account.
 
     ```bash
     az storage account show -n mystorageaccountname --query 'primaryEndpoints.blob' -otsv
     ```
 
-2. Retrieve the shared access signature for the storage account. For example:
+2. Retrieve the shared access signature for the storage account.
 
     ```bash
     az storage container generate-sas --account-name <storage-account> --name <container> --permissions acdlrw --expiry <date-time> --auth-mode key
     ```
 
-3. Create a Kubernetes secret to hold the storage account key. For example:
+3. Create a Kubernetes secret to hold the storage account key.
 
     ```bash
+    export SAS_TOKEN=$(az storage container generate-sas --account-name <storage-account> --name <container> --permissions acdlrw --expiry <date-time> --auth-mode key)
     kubectl create secret generic my-azure-storage-credentials \
-      --from-literal "sas=$(az storage container generate-sas --account-name <storage-account> --name <container> --permissions acdlrw --expiry <date-time> --auth-mode key)"
+      --from-literal "shared-access-key=$SAS_TOKEN"
     ```
 
-4. Configure `azure` artifact as follows in the yaml.
+4. Configure `azure` artifact as follows in the YAML.
```

Comment:
```suggestion
4. Configure an `azure` artifact:
```

Should match the changes in the Access Key section from https://github.com/argoproj/argo-workflows/pull/13376#discussion_r1687079403